client/style.css

```diff
@@ -416,17 +416,25 @@ h1{
   float: right;
 }
 
-.add-question-form label {
+.add-question-form label, .add-answer-form label {
```

Comment:
I like this reusing of the style attributes for identical classes. I'll have to include that in future css work myself!